test/e2e/discovery/service_discovery.go

```diff
@@ -304,7 +304,7 @@ func verifyServiceIpWithDig(f *framework.Framework, srcCluster, targetCluster fr
 		return stdout, nil
 	}, func(result interface{}) (bool, string, error) {
 		doesContain := strings.Contains(result.(string), serviceIP)
-		By(fmt.Sprintf("Validating that dig result %s %q", op, result))
+		By(fmt.Sprintf("Validating that dig result %s %q", op, serviceIP))
```

Comment:
```suggestion
		By(fmt.Sprintf("Validating that dig result %s %q %s", result, op, serviceIP))
```

Reply:
We need to capture result too. It lets us know what is the actual result returned, which is useful when we run into test failures.

Reply:
Yeah, it will be helpful.